src/Dotnet.Script.Tests/ScriptExecutionTests.cs

```diff
@@ -233,6 +235,17 @@ public void ShouldHandleNonExistingRemoteScript()
         }
 
         [Fact]
+        public void ShouldHandleScriptUsingTheProcessClass()
+        {
+            // This test ensures that we can load the Process class.
+            // This used to fail when executing a netcoreapp2.0 script
+            // from dotnet-script built for netcoreapp2.1
+            var result = Execute(Path.Combine("Process", "Process.csx"));
+            Assert.Contains("Success", result.output);
+        }
+
+
+        [Fact(Skip = "This also failes when run a standard netcoreapp2.1 console app")]
```

Comment:
we could still run this at least on netcoreapp2.0? Something like this should do:

```
    public class OnlyOnNetCore20FactAttribute : FactAttribute
    {
        public OnlyOnNetCore20FactAttribute()
        {
                Skip = RuntimeHelper.TargetFramework != "netcoreapp2.0";
        }
    }
```